test-built

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 
-cd dist
-if [ ! -d bundle ]
```

Comment:
Oh, I see, that makes sense.  In that case the title of the `build` script has kind of lost its meaning, so how about we rename it to `predeploy` or inline it in the `deploy` script?